src/server/auth.ts

```diff
@@ -73,17 +73,7 @@ export const authOptions: NextAuthOptions = {
       return session;
     },
 
-    async jwt({ token, trigger, session }) {
-      if (trigger === "update") {
-        const newToken = {
-          ...token,
-          ...session?.user,
-          picture: session?.user?.image || "",
-        };
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-        return newToken;
-      }
-
```

Comment:
we have to pass the profile picture url through jwt callback and session callback.